src/material/core/typography/_all-typography.scss

```diff
@@ -45,10 +47,97 @@
 //   (where define-typography-config is defined) because putting it there would cause a circular
 //   dependency with mdc-helpers. We should refactor so these can live in the same place.
 
+// Converts a map containing rem values to a map containing px values.
+@function rem-to-px($x, $px-per-rem: 16px) {
```

Comment:
Should this be underscored so it can't be imported in other files?

Reply:
Done, though it wasn't exported by the index file anyways